pkg/beacon/node.go

```diff
@@ -75,7 +76,11 @@ func (n *node) JoinDKGIfEligible(
 		return
 	}
 
-	logger.Infof("selected group members = %s", selectedOperators)
+	logger.Infof(
+		"selected group members for DKG with seed [0x%x] = %s",
+		dkgSeed,
+		selectedOperators,
+	)
```

Comment:
Instead of this, I think we should use `dkgLogger` which is defined on line 71. It is adding seed context, and we can filter all DKG executor messages by that context.

Reply:
Awesome suggestion, thanks! b396cc2e1abca2251b798d225a423689b5c6430d